conf.d/20_output_journald_elasticsearch.conf

```diff
@@ -34,7 +34,6 @@ output {
     elasticsearch {
       id => "journald_es"
       index => "journald-%ELASTICSEARCH_INDEX_SUFFIX%%{+YYYY.MM.dd}"
-      document_type => "%{transport}"
```

Comment:
We should capture transport and store it in another field for reference, now that we drop `document_type`.

Reply:
Agreed that this is not required